base/threadingconstructs.jl

```diff
@@ -1,7 +1,6 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
-export threadid, nthreads, @threads, @spawn,
-       threadpool, nthreadpools
+export threadid, @threads, @spawn, threadpool, nthreadpools
```

Comment:
`nthreads` is deprecated but should still be exported, otherwise it would be a breaking change?
Also, will this work given that the deprecation is `@eval`-ed?
```suggestion
export threadid, nthreads, @threads, @spawn, threadpool, nthreadpools
```

Reply:
There is problems in [SparseArrays]( https://github.com/JuliaSparse/SparseArrays.jl/blob/2bbdd7a12ead8207593655c541ba347761a9c663/src/solvers/cholmod.jl#L224), so I couldn't make it a proper deprecation yet until that package is fixed

Reply:
We need to put this back until we can do a proper deprecation though.

Reply:
Re. The SparseArrays issue https://github.com/JuliaSparse/SparseArrays.jl/pull/204

Reply:
See https://github.com/JuliaSparse/SparseArrays.jl/pull/206

Reply:
SparseArrays is now fixed on julia master